### ISSUE TITLE
Dice as metric

### ISSUE THREAD

**Issue by alqurri77** [qubvel/segmentation_models.pytorch#590]
Hi;

I was looking to see if metrics implement Dice, because I'm interested to use it. But I couldn't find it. (I found Dice as loss but not as metric).

Thank you!

**Comment by alqurri77** [comment]
I'm not sure how without almost do everything from scratch. I'm looking for some thing simple like what is shown on this link:

[https://torchmetrics.readthedocs.io/en/stable/pages/overview.html](https://torchmetrics.readthedocs.io/en/stable/pages/overview.html)
```

train_accuracy = Accuracy()
valid_accuracy = Accuracy()

for epoch in range(epochs):
    for x, y in train_data:
        y_hat = model(x)

        # training step accuracy
        batch_acc = train_accuracy(y_hat, y)
        print(f"Accuracy of batch{i} is {batch_acc}")

    for x, y in valid_data:
        y_hat = model(x)
        valid_accuracy.update(y_hat, y)

    # total accuracy over all training batches
    total_train_accuracy = train_accuracy.compute()

    # total accuracy over all validation batches
    total_valid_accuracy = valid_accuracy.compute()

    print(f"Training acc for epoch {epoch}: {total_train_accuracy}")
    print(f"Validation acc for epoch {epoch}: {total_valid_accuracy}")

    # Reset metric states after each epoch
    train_accuracy.reset()
    valid_accuracy.reset()
```
